R/quboleCommand.R

```diff
@@ -0,0 +1,152 @@
+library(R6)
+QuboleCommand <- R6Class("QuboleCommand",public = list(
+  api_token = NULL,
+  poll_interval = 5,
+  initialize = function(api_token,poll_interval)
+    {
+    if (!missing(api_token)) self$api_token <- api_token
+    if (!missing(poll_interval)) self$poll_interval <- poll_interval
+    rPython::python.load(file.path(find.package("qds"),"exec","r-quboleCommand.py" ))
+    rPython::python.call("connect",self$api_token,self$poll_interval)
+    },
+  #' Prints command object for a given commandId
+  #' @method 
+  check = function(commandId)
+  {
+    rPython::python.call("check",commandId)
```

Comment:
Changing "check" method to return, so that user can implement their logic to wait and get the result later. Else it returns null and may not be of much use to the user